### ISSUE TITLE
luci-mod-status: Cannot read properties of undefined (reading 'rx_bytes')

### ISSUE THREAD

**Issue by pameruoso** [openwrt/luci#6492]
## Steps to reproduce:

1. go to: Openwrt -> Status or simply click on the Openwrt logo in the main page.

## Actual behavior:
![image](https://github.com/openwrt/luci/assets/42093926/299bf869-aeb7-40e5-a8d3-c4cc108351e9)
![image](https://github.com/openwrt/luci/assets/42093926/09ce667f-c975-425e-b72f-80fc3c60a5db)

## Expected behavior:
The status page should load correctly

**Example: Replace the following lines and remove this**
```
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='23.05.0-rc2'
DISTRIB_REVISION='r23228-cd17d8df2a'
DISTRIB_TARGET='mediatek/filogic'
DISTRIB_ARCH='aarch64_cortex-a53'
DISTRIB_DESCRIPTION='OpenWrt 23.05.0-rc2 r23228-cd17d8df2a'
DISTRIB_TAINTS=''
```

**Comment by pameruoso** [comment]
@jow- is this something you pushed on the 27th of July?

**Comment by pameruoso** [comment]
nevermind . fixed with 00a941e
